.github/Pull_Request_and_Code_Review_Standards.md

```diff
@@ -0,0 +1,41 @@
+# Pull Request and Code Review Standards
+
+-   Unless it's a trivial change (spelling mistake, code formatting, etc.), there should be an existing issue that will be closed by the given PR.
+
+-   The branch name from which PR is being made should follow the convention: `\<issue_number\>-short-description` (e.g. \`186_implement_data_extract\`).
+
+-   The title for the PR should provide a brief description about the changes.
+
+-   The PR should be linked to the issue(s) that it targets, either by using [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) (like `fixes`, `closes`, etc.) in the PR description or by using `development` feature.
+
+-   Even if the PR doesn't close any issues, it should refer to the comment or discussion that motivated it. Lacking any of these, the PR description should be self-sufficient to explain the motivation for the PR.
+
+-   If the PR is a work in progress, you can convert it to a draft PR and add `WIP` to the PR title.
+
+-   Reviewers should not conduct code review of draft PRs, unless otherwise requested by the contributors.
+
+-   Once the PR is marked as "Ready for Review", contributors should remember to remove `WIP` from the PR title.
+
+-   Before marking the PR as "Ready for Review", contributors should also make sure that they are following the tidyverse style guide (enforced via `{styler}` package) and to make sure that CI/CD pipelines are executing successfully. If this is not possible for some reason (e.g. because the code in *other repositories* must be changed) - the contributor should create new issue(s) for those required code changes, link them in the PR, and shortly explain why the change in the PR itself is not possible.
```

Comment:
```suggestion
-   Before marking the PR as "Ready for Review", contributors should also make sure that they are following the tidyverse style guide (enforced via `{styler}` package) and to make sure that automatic GitHub tests all finish successfully. If it is not possible for tests to finish successfully for some reason (e.g., because the PR relies on changes in *other repositories*), then the contributor should create new issue(s) for those required code changes, link them in the PR, and shortly explain why the change in the PR itself is not possible.
```